app/models/concerns/routing.rb

```diff
@@ -0,0 +1,10 @@
+module Routing
```

Comment:
To allow us to use `discussion_url` without manually passing a host in the serializer